qemu/tests/cfg/sev_basic_config.cfg

```diff
@@ -0,0 +1,16 @@
+- sev_basic_config:
+    type = sev_basic_config
+    kill_vm = yes
+    login_timeout = 240
+    image_snapshot = yes
```

Comment:
this param may be not required, in case you want a snapshot on top of image1

Reply:
if not adding image_snapshot, boot multi vms in the same time will meet all VMs use the same image and cannot boot issue.

Reply:
Actually, this param was introduced in order to keep compatibility of '-drive snaphot=on'(like a workaround), which is not supported by '-blockdev', we can use clone_master = yes, master_images_clone = image1 for multiple vms, but I found that many cases are using it for multiple VMs purpose, so it's OK for me. 

Thanks for your comments, too.